app/app.h

```diff
@@ -4,5 +4,38 @@
 
 namespace vsite::oop::v5 
 {
+	double operator"" _inch(unsigned long long v);
+	double operator"" _eur(unsigned long long v);
 
+	class tire {
+		double m_diameter;
+	public:
+		explicit tire(double diameter_cm);
+		tire() = delete;
```

Comment:
Zašto biste `delete`-ali default konstruktor kad ste već napisali konstruktor s argumentima (i automatski više nemate generirani default konstruktor)?